### PR TITLE
Pulssi 954 debian package

### PR DESCRIPTION
**Pull request by kompa3**
Added debian packaging 1) for customers, and 2) to enable CoreOS monitoring that's based on debian image

### CODE REVIEW

**Comment by kompa3** [comment]
I already built debian packages, tested them and added them to https://github.com/digiapulssi/zabbix-agent/releases.

File: debian/Dockerfile.debian8 [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM debian:8
```
Comment (sapajune) [inline debian/Dockerfile.debian8]: I assume this does not use the slim distribution as base which would make the container remarkably smaller.

Comment (kompa3) [reply debian/Dockerfile.debian8]: The dockerfile is used solely as a build environment for the deb package, so it doesn't matter if the container is big or small

**Comment by kompa3** [comment]
The dockerfiles are separate. These dockerfiles are used to generate .deb packages. In docker-host-monitoring branch, the deb building is modified so that it takes Zabbix agent sources from zabbix/docker-host-monitoring branch - ie. the host monitoring patches.

**Pull request merged**